Source/Device.generated.swift

```diff
@@ -721,7 +728,7 @@ public enum Device {
     public var has3dTouchSupport: Bool {
       return isOneOf(Device.allDevicesWith3dTouchSupport)
     }
-  
+    
```

Comment:
Please remove the trailing whitespace. You might also have to change this in the gyb file.

Reply:
This one still stands

Reply:
That code is outdated. If you look at the code itself there is no space.